app/src/main/java/com/isc/hermes/SignUpActivityView.java

```diff
@@ -86,16 +88,27 @@ protected void onStart() {
      *
      * @param view it contains the event info.
      */
-    public void SignUp(View view) {
+    public void signUp(View view) {
         if (authenticator != null) return;
         authenticator = authenticationServices.get(view.getId());
         if (authenticator == null) return;
-        startActivityForResult(
+        startActivityForResult( //TODO Solve This
```

Comment:
Please be more descriptive with this TODO.

Reply:
Done